cmd/antrea-agent/options.go

```diff
@@ -173,6 +173,9 @@ func (o *Options) setDefaults() {
 	} else {
 		o.setExternalNodeDefaultOptions()
 	}
+	if o.config.Multicluster.Namespace == "" {
```

Comment:
This should be moved to setK8sNodeDefaultOptions(), and we should first check Multicluster feature gate is enabled. Better to check enableGateway too if it is needed only when Gateway is enabled.

Reply:
Done